CMakeLists.txt

```diff
@@ -1,79 +1,72 @@
-cmake_minimum_required(VERSION 3.10)
-
-#
-# Choosing to build the unittest (Catkin vs Modern CMake vs Ament).
+# Copyright 2024 PAL Robotics SL. All Rights Reserved
 #
-if(DEFINED CATKIN_ENABLE_TESTING)
-  set(BUILD_TESTING ${CATKIN_ENABLE_TESTING})
-endif()
+# Unauthorized copying of this file, via any medium is strictly prohibited,
+# unless it was supplied under the terms of a license agreement or nondisclosure
+# agreement with PAL Robotics SL. In this case it may not be copied or disclosed
+# except in accordance with the terms of that agreement.
```

Comment:
Are you sure about this license here ?

Reply:
https://github.com/ros-controls/ros2_control/blob/0433960580c5834b11af9703e3e34ace86824d01/joint_limits/include/joint_limits/joint_limits_urdf.hpp#L4

This seems more appropriate for an opensource code.

Reply:
Bad copy-paste I will remove it all together as the license is in the LICENSE file already.